tgui/packages/tgui/interfaces/CharacterPreferences/type.ts

```diff
@@ -0,0 +1,79 @@
+export interface CharacterPreferencesData {
```

Comment:
I would imagine a bunch of these are probably better represented as `BooleanLike`?

Reply:
Or I guess `string | BooleanLike`, which is to stop TypeScript complaining it seems?